requirements.txt

```diff
@@ -1,9 +1,9 @@
 PyYAML==3.11
 cached-property==1.2.0
-docker-py==1.7.0
 dockerpty==0.4.1
 docopt==0.6.1
 enum34==1.0.4
+git+https://github.com/docker/docker-py.git@master#egg=docker-py
```

Comment:
Oh, I missed this before, please pin to a git sha instead of a branch. That way we don't accidentally pick up other docker-py changes.